iot-hub/Samples/service/AutomaticDeviceManagementSample/AutomaticDeviceManagementSample.cs

```diff
@@ -25,44 +25,47 @@ public AutomaticDeviceManagementSample(RegistryManager registryManager)
         public async Task RunSampleAsync()
         {
             Console.WriteLine("Create configurations");
-            await AddDeviceConfiguration("config001").ConfigureAwait(false);
-            await AddDeviceConfiguration("config002").ConfigureAwait(false);
-            await AddDeviceConfiguration("config003").ConfigureAwait(false);
-            await AddDeviceConfiguration("config004").ConfigureAwait(false);
-            await AddDeviceConfiguration("config005").ConfigureAwait(false);
+
+            // save unique config names to be used for deletion
+            var configs = new List<string>(5);
+            for (int i = 0; i < 5; i++)
+            {
+                configs.Add($"config00{i+1}_{Guid.NewGuid()}");

```

Comment:
```suggestion
                configs.Add($"config00{i}_{Guid.NewGuid()}");

```
How about we index starting from 000?

Reply:
Fine with me, this was just making it as close as possible to the original naming scheme. I agree indexing from zero is cleaner.